src/View/Helper/ListsHelper.php

```diff
@@ -337,9 +337,21 @@ public function displayVisibilityOption($listId, $value)
             ?>
             <input type="radio"  name="visibility" data-list-id='<?= $listId ?>'  value="{{visibility}}" checked hidden ng-init="visibility = '<?= $value ?>';"/>
             <md-radio-group ng-controller='optionsCtrl' ng-model='visibility' ng-change='visibilityChanged()'>
-                <md-radio-button value='public' class='md-primary'>
+                <md-radio-button value='public' class='md-primary' title=
+                                 '<?=h(__("The list will be shown on the sentences page"))?>'>                  
```

Comment:
I think that tooltip should rather be more like `The list is displayed everywhere for everyone`, i.e. we shouldn't tie ourselves to future problems where we may want to display the list on pages other than the sentence page too.